MCOL-3577

benthompson15 opened the pull request.

benthompson15 committed:
MCOL-3577: Add functionality to sync S3 storage for suspendDatabaseWrites.

benthompson15 committed:
MCOL-3577: Found an issues with the order of dropPrefix Cache before Sync.

benthompson15 committed:
MCOL-3577: cleanup debug logging and comments from copy/paste.

pleblanc1976 commented:
Gave Ben feedback in person.  As is it will only sync the node it's running on, and on the SM side, the original syncNow(prefix) can deadlock, and should just call his new syncNow() (everything) fcn for now.

benthompson15 committed:
MCOL-3577: Make this run syncFS for storagemanager on all nodes when suspend writes is done from mcsadmin.

benthompson15 committed:
MCOL-3577: update syncNow with prefix
to behave like syncNow no prefix.
This should be corrected eventually but is safer now as
previous implementation would possibly result in deadlock.

benthompson15 committed:
MCOL-3577: Fix messaging for filesytem sync.

pleblanc1976 reviewed (approved):
Hard to say whether those procmon/mgr are right.  Looks OK, but will have to rely on your testing.
